study.md

```diff
@@ -111,15 +117,18 @@ The array should contain two strings, "drinking beer" and "propane and propane
 accessories".
 
 ```javascript
-// your answer here
+hank[middleName] = "Rutherford";
```

Comment:
This will assume that `middleName` is a variable and if it doesn't exist, then it will throw an error.